app/controllers/citizens/accounts_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Citizens
+  class AccountsController < ApplicationController
+    before_action :authenticate_applicant!
```

Comment:
😕 I'm a bit confused as how we're authenticating the applicant here, since our app has (**or more accurately** will have) authentication for the applicant which is _supposed_ to be completely independent from the applicant providing us access to his/her bank account through TrueLayer.

Reply:
I've updated the controller at the start of the Citizen journey so that it signs the applicant into Devise and stores the current applicant details in session. It is then these credentials that are used to get the correct `applicant` when the user returns from TrueLayer.